app/public/static/js/components/keypad.js

```diff
@@ -91,18 +92,25 @@ AFRAME.registerComponent("keypad", {
       this.solvedPuzzleEntity.setAttribute("visible", "true");
     } else {
       // Not solved yet
+      keypad_label = data.hasOwnProperty("keypad_text")
+        ? data.keypad_text
+        : "Enter Password";
       el.setAttribute("super-keyboard", {
-        label: "Enter Password",
+        label: keypad_label,
         labelColor: "black",
```

Comment:
Can we make this customizable? Or at the very least default to a non-black colour, it's a bit hard to see imo

Reply:
Made the default `white` and also customizable